src/axom/primal/operators/clip.hpp

```diff
@@ -129,6 +129,37 @@ AXOM_HOST_DEVICE Polyhedron<T, 3> clip(const Octahedron<T, 3>& oct,
   return detail::clipOctahedron(oct, tet, eps);
 }
 
+/*!
+ * \brief Clips a 3D tetrahedron against another tetrahedron in 3D, returning
+ *        the geometric intersection as a polyhedron
+ *
+ *  This function clips the first tetrahedron by the 4 planes obtained from the
+ *  second tetrahedron's faces (normals point inward). Clipping the
```

Comment:
Is this normal direction the convention that is used in other similar methods in the code?

Reply:
> Is this normal direction the convention that is used in other similar methods in the code?

The normal direction used here is as-described in the [Planes implementation](https://github.com/LLNL/axom/blob/268b09ab46c43640c826a701688b58d4ff8a1e39/src/axom/primal/geometry/Plane.hpp#L67-L96).

So the normals pointing inwards means that the inside of the tetrahedron are in the positive half space above the planes.

This is consistent with the Planes convention of normals, and I think what's described [here](https://github.com/LLNL/axom/blob/268b09ab46c43640c826a701688b58d4ff8a1e39/src/axom/primal/operators/orientation.hpp#L32-L49) about determining the orientation of primitives.